tests/integration/cli/test_tabular_format.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os.path as osp
+
+import pytest
+
+from datumaro.components.dataset import Dataset
+from datumaro.plugins.data_formats.tabular import *
+
+from ...requirements import Requirements, mark_requirement
+
+from tests.utils.assets import get_test_asset_path
+from tests.utils.test_utils import TestCaseHelper, TestDir, compare_datasets
+from tests.utils.test_utils import run_datum as run
+
+
+@pytest.fixture()
+def fxt_tabular_root():
+    yield get_test_asset_path("tabular_dataset")
+
+
+@pytest.fixture()
+def fxt_electricity_path(fxt_tabular_root):
+    yield osp.join(fxt_tabular_root, "electricity.csv")
+
+
+@pytest.fixture()
+def txf_electricity(fxt_electricity_path):
+    yield Dataset.import_from(fxt_electricity_path, "tabular")
+
+
+@pytest.fixture()
+def fxt_buddy_path(fxt_tabular_root):
+    yield osp.join(fxt_tabular_root, "adopt-a-buddy")
+
+
+@pytest.fixture()
+def fxt_buddy_target():
+    yield ["breed_category", "pet_category"]
```

Comment:
```suggestion
    yield ("breed_category", "pet_category")
```
Using a generator may efficient than using a list when you return by `yield`
https://medium.com/free-code-camp/python-list-comprehensions-vs-generator-expressions-cef70ccb49db